inference-api/python-client/client.py

```diff
@@ -0,0 +1,91 @@
+import requests
+
+from asgiref.sync import async_to_sync
+from retry import retry
+from typing import Optional
+
+from .exceptions import InvalidInferenceAPIClientArgsException
+from .types import Inference
+from .django_jwt_client import DjangoJWTClient
+
+class InferenceAPIClient:
+  AUTH_PATH = '/api/auth/jwt_login'
+  GET_INFERENCE_PATH = 'models/{model_id}/inferences/{study_instance_uid}/{series_instance_uid}'
+
+  def __init__(
+    self,
+    private_key_filename: Optional[str] = None,
+    private_key_string: Optional[str] = None,
+    base_url: str = 'https://api.bunkerhillhealth.com/',
+  ) -> None:
+    self._validate_args(private_key_filename, private_key_string)
+    if private_key_filename:
+      with open(private_key_filename, 'r') as f:
+        client_private_key = f.read()
+    elif private_key_string:
```

Comment:
Changed to using `else` , and also changed the logic everywhere to use `if varname` rather than `if varname is not None`, so that "" is treated the same as None, which may be more in line with user expectations.